src/sql.rs

```diff
@@ -618,7 +619,26 @@ async fn maybe_add_from_param(
         .query_map(
             query,
             paramsv![],
-            |row| row.get::<_, String>(0),
+            //|row| row.get::<_, String>(0), // <=== this crashes when non utf8 byte in column
+
+            // cs: maybe the following could be done even more simple than here?
+            |row| {
+                let value = match row.get_raw(0) {
+                    ValueRef::Text(t) => {
+                        // changed due to clippy complain
+                        //~let col_str = String::from_utf8_lossy(t).to_string();
+                        //~info!(context, "maybe_add_from_param() - col_str: \"{}\"", col_str);
+                        //~col_str
+                        String::from_utf8_lossy(t).to_string()
+                    }
+                    _ => {
+                        let s_err = "maybe_add_from_param() - error - get_raw(0)".to_string();
+                        info!(context, "{}", s_err);
+                        s_err
```

Comment:
This will result in searching for file with a name `maybe_add_from_param...`.

Reply:
Error text should point to the function where it happend.
But I think `info!` should be a `warn!`